src/components/Sykmelding/ActionSection/ActionSection.tsx

```diff
@@ -83,4 +93,6 @@ function MutationResultFeedback({
     return result.error ? <Alert variant="error">Kunne ikke registrere sykmeldingen</Alert> : <>{children}</>;
 }
 
+export { useHandleRegister } from './useHandleSave';
```

Comment:
Er det ein grunn til at `useHandleRegister` blir exportert her i staden for å importere direkte i `SykmeldingForm`?